apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page.tsx

```diff
@@ -191,38 +196,45 @@ const Page: NextPageWithLayout = () => {
   const showEditBar =
     isDirty || !!draftLinksToCreate.length || !!draftLinksToArchive.length;
 
+  const draftEntity = getRoots(draftEntitySubgraph)[0];
+  const isQueryEntity =
+    draftEntity?.metadata.entityTypeId === QUERY_ENTITY_TYPE_ID;
+
+  const entityEditorProps: EntityEditorProps = {
+    draftLinksToCreate,
+    setDraftLinksToCreate,
+    draftLinksToArchive,
+    setDraftLinksToArchive,
+    entitySubgraph: draftEntitySubgraph,
+    readonly,
+    refetch,
+    setEntity: (changedEntity) => {
+      setIsDirty(true);
+      updateEntitySubgraphStateByEntity(changedEntity, setDraftEntitySubgraph);
+    },
+  };
```

Comment:
I think you can revert this change, unless there is some benefit to keeping it

Reply:
Removed both here https://github.com/hashintel/hash/commit/5e1677d5e862f76ba67151b89393e840978a79a0